gtsfm/utils/io.py

```diff
@@ -101,6 +101,19 @@ def save_json_file(
         json.dump(data, f, indent=4)
 
 
+def read_json_file(fpath: Union[str, Path]) -> Any:
+    """Load dictionary from JSON file.
+
+    Args:
+        fpath: Path to JSON file.
+
+    Returns:
+        Deserialized Python dictionary or list.
+    """
+    with open(fpath, "rb") as f:
```

Comment:
should we read/write in plaintext?

Reply:
plaintext should be sufficient. i'll update it.